src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -10,6 +10,10 @@ public class RobotConstants {
 
     public static final DriverStation.Alliance SIMULATION_ALLIANCE = DriverStation.Alliance.Blue;
 
+    public static final boolean IS_RUNNING_ON_USB_B = false;
```

Comment:
what about simulation what i am suppose to do?

Reply:
nothing. this is only relevant when running on robot

Reply:
I know but I doesn’t tell a lot like that, if I didn’t know the purpose of that I would consider what I am suppose to insert there.
But it doesn’t matter, u need to delete it and use args

Reply:
I think that switching this to work with args is for another PR since there is a branch that added this functionallity

Reply:
We’ll see what’s ready to merge first, if this pr ok else add it after the gradle merge